repos/fdbt-site/src/pages/api/managePurchaseMethod.ts

```diff
@@ -26,25 +31,26 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
         } = req.body;
 
         const id = req.body.id && Number(req.body.id);
+        const isCapped = req.body.isCapped === 'true' ? true : false;
```

Comment:
```suggestion
       const isCapped = req.body.isCapped === 'true';
```

Reply:
I added this comment previously - please can we implement it?

Reply:
Removed as it does not make any sense because we are just returning and the expresssion will be doing the same.

Reply:
The code needs changing because as it currently stands, its doing this: 

`if (true) {
return true} else {
return false}`

req.body.isCapped === 'true' is **already** a boolean, which will equate to true or false. The code `? true : false; ` is doing nothing.